app/src/org/commcare/activities/CommCareSetupActivity.java

```diff
@@ -888,6 +888,7 @@ private boolean isSingleAppBuild() {
     public void onRequestPermissionsResult(int requestCode,
                                            @NonNull String[] permissions,
                                            @NonNull int[] grantResults) {
+        super.onRequestPermissionsResult(requestCode, permissions, grantResults);

```

Comment:
Thinking this should be okay but just want to make sure it wasn't left out on purpose since this is traditional CC code. @shubham1g5?

Reply:
I don't think this will cause any issues in the future. The code was showing an error, which is why I resolved it. However, if you still want to remove this line, I can do that.